CHANGELOG.md

```diff
@@ -35,4 +36,3 @@
   the root of a domain (#643)
 - Aggregate over connection types in the cred explorer (#502)
 - Start tracking changes in `CHANGELOG.md`
-
```

Comment:
👾

Reply:
In the future, please make extraneous changes like this one in a
separate pull request.